src/ui/settings/ViewSubscriptionManagement/ViewSubscriptionManagement.qml

```diff
@@ -152,15 +152,18 @@ VPNFlickable {
     function populateListModels() {
         // Subscription info model
         // Subscription plan
-        subscriptionInfoModel.append({
-            _objectName: "subscriptionItem-plan",
-            labelText: VPNl18n.SubscriptionManagementPlanLabel,
-            valueText: getPlanText(
-                VPNSubscriptionData.planCurrency,
-                VPNSubscriptionData.planAmount,
-            ),
-            type: "text",
-        });
+
+        if (VPNSubscriptionData.type !== VPNSubscriptionData.SubscriptionApple) {
```

Comment:
instead of doing this, can you do:
`if (VPNSubscriptionData.planCurrency &&VPNSubscriptionData.planAmount) {`